src/library/Uno.Material/Styles/Controls/v2/Flyout.xaml

```diff
@@ -0,0 +1,650 @@
+
+	<x:String x:Key="M3MaterialFlyoutCheckGlyphPathStyle">M30.561941,0L31.997,1.393004 10.467954,23.597999 0,15.350999 1.2379759,13.780992 10.287961,20.909952z</x:String>
+	<x:String x:Key="M3MaterialFlyoutRightArrowPathStyle">M0,0L25.194,16 0,32z</x:String>
+	<x:Double x:Key="M3MaterialFlyoutPresenterMinWidth">112</x:Double>
+	<x:Double x:Key="M3MaterialFlyoutPresenterMaxWidth">280</x:Double>
+	<x:Double x:Key="M3MaterialFlyoutMenuItemHeight">48</x:Double>
+	<x:Double x:Key="M3MaterialFlyoutMenuItemIconHeight">16</x:Double>
+	<x:Double x:Key="M3MaterialFlyoutMenuItemIconWidth">16</x:Double>
+	<x:Double x:Key="M3MaterialFlyoutMenuToggleCheckGlyphWidth">16</x:Double>
+	<x:Double x:Key="M3MaterialFlyoutMenuToggleCheckGlyphHeight">16</x:Double>
+	<x:Double x:Key="M3MaterialFlyoutMenuSeparatorHeight">1</x:Double>
+	<x:Double x:Key="M3MaterialFlyoutFontSize">14</x:Double>
+	<x:Double x:Key="M3MaterialFlyoutCharacterSpacing">7.143</x:Double>
+	<GridLength x:Key="M3MaterialFlyoutMenuItemRightMargin">38</GridLength>
+
+	<!--  Material FlyoutPresenter Style  -->
+	<Style x:Key="M3MaterialFlyoutPresenterStyle"
+	       TargetType="FlyoutPresenter">
+		<Setter Property="HorizontalAlignment"
+		        Value="Stretch" />
+		<Setter Property="VerticalAlignment"
+		        Value="Stretch" />
+		<Setter Property="HorizontalContentAlignment"
+		        Value="Stretch" />
+		<Setter Property="VerticalContentAlignment"
+		        Value="Stretch" />
+		<Setter Property="MinWidth"
+		        Value="NaN" />
+		<Setter Property="MinHeight"
+		        Value="NaN" />
+		<Setter Property="MaxWidth"
+		        Value="NaN" />
+		<Setter Property="MaxHeight"
+		        Value="NaN" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="FlyoutPresenter">
+					<Border HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+					        VerticalAlignment="{TemplateBinding VerticalAlignment}">
+						<Border.Transitions>
+							<TransitionCollection>
+								<PaneThemeTransition Edge="Bottom" />
+							</TransitionCollection>
+						</Border.Transitions>
+						<ContentPresenter HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+						                  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+						                  Content="{TemplateBinding Content}"
+						                  ContentTemplate="{TemplateBinding ContentTemplate}"
+						                  ContentTransitions="{TemplateBinding ContentTransitions}" />
+					</Border>
+				</ControlTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>
+
+	<!--  Material Content FlyoutPresenter Style  -->
+	<Style x:Key="M3MaterialContentFlyoutPresenterStyle"
+	       TargetType="FlyoutPresenter">
+
+		<Setter Property="Foreground"
+		        Value="{StaticResource OnSurfaceBrush}" />
+		<Setter Property="Background"
+		        Value="{StaticResource SurfaceBrush}" />
+		<!--  Start: Label Large Typo  -->
+		<Setter Property="FontSize"
```

Comment:
maybe move them into?: Styles\v2\SharedTypography.xaml: M3MaterialLargeLabel(FontSize|FontFamily|...)

Reply:
Yeah I've been thinking about that too, https://github.com/unoplatform/Uno.Themes/pull/743#discussion_r851520040. 

There's no way to create something like a StaticResource that is just a collection of Setters, like without the Style and TargetType? That would be cool

Reply:
Hmm the FontSize/Family/Character Spacing/etc would be different for each design system. So things are going to get messy if you have more than one design system's resources loaded.

Reply:
cant do that with setters, but you could (like you just did) with values
\---
isnt that what the prefix part ("M3Material") is for? so they dont collide.

Reply:
plus, this is a material flyout, it should just use tje material typography